src/components/common/NameRequestInfo.vue

```diff
@@ -110,7 +110,7 @@
             sm="3"
             class="pr-4"
           >
-            <label>Name</label>
+            <label>Resulting Business Name</label>
```

Comment:
This is a common component, so your changes here affect other filings.

I don't fancy the idea of a duplicate component for amalgamation filings, and I don't love the idea of adding `v-if="isAmalgamationFiling"` in a common component (as this adds special cases), but I think that will be easiest and cleanest way to go for now.